### DISCUSSION TITLE
Pick & place file

### DISCUSSION THREAD

**Discussion by Mannlig** [PricelessToolkit/SOILSENS-V5W#5] [Q&A]
Hey, 
have allready ordered a few wireless sensors form your website and since im impatiant of nature i was looking in try making your project with pcbway, but im having a hard time with a pick and place file. 
do you have one that you can share?

**Comment by PricelessToolkit** [comment]
Hi, Thank you for your support!!, Sorry, but I don't share PnP files. Please provide your order number through the shop's contact form, and I will check when your order will be ready.
